### PR TITLE
feature/SU-47 - Campos creación Aliados

### PR DESCRIPTION
**Pull request by dstanca**
feature/SU-47 - Campos creación Aliados

### CODE REVIEW

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200400_SU_47_code_value_tipo_cuenta_bancaria_INSERT.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="fineract" id="1" >
+        <insert tableName="m_code">
+            <column name="code_name" value="Tipo cuenta bancaria"/>
+            <column name="is_system_defined" valueBoolean="false"/>
+        </insert>
```
Comment (fiter-julius-oketayot) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200400_SU_47_code_value_tipo_cuenta_bancaria_INSERT.xml]: I have noticed an inconsistent behaviors especially in the datatable module when using using m_code value with spaces
e.g. Tipo cuenta bancaria could be changed to TipoCuentaBancaria

**Review by fiter-julius-oketayot — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200500_SU_47_code_value_perfil_tributario_INSERT.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="fineract" id="1" >
+        <insert tableName="m_code">
+            <column name="code_name" value="Perfil tributario (regimen iva)"/>
```
Comment (fiter-julius-oketayot) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200500_SU_47_code_value_perfil_tributario_INSERT.xml]: Perfil tributario (regimen iva) This code name might bring issue as well

**Comment by fiter-julius-oketayot** [comment]
The PR should target fiter/sumas/dev branch, right??
@Napho

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200400_SU_47_code_value_tipo_cuenta_bancaria_INSERT.xml (hunk 1 above)
Comment (dstanca) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200400_SU_47_code_value_tipo_cuenta_bancaria_INSERT.xml]: Agreed!
Will change it.

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200500_SU_47_code_value_perfil_tributario_INSERT.xml (hunk 2 above)
Comment (dstanca) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/20240308200500_SU_47_code_value_perfil_tributario_INSERT.xml]: Agreed as well!

**Pull request closed without merging**